packages/realm/bindgen/src/templates/jsi.ts

```diff
@@ -239,6 +239,9 @@ function convertPrimToJsi(addon: JsiAddon, type: string, expr: string): string {
     case "uint64_t":
       return `bigIntFromU64(_env, ${expr})`;
 
+    case "std::chrono::milliseconds":
+      return `bigIntFromU64(_env, std::chrono::milliseconds(${expr}).count())`;
```

Comment:
Thinking more about this change. We should have been able to ignore this primitive all together, by simply not including it in the `js_opt_in_spec.yml`. Did bindgen fail if you didn't put this code in? If not - perhaps we should extend the "opt-in" feature to include primitives? What do you think @elle-j?